AutocompleteClient/FW/Logic/Request/CIOPurchaseItem.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CIOPurchaseItem.swift
+//  Constructor.io
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+public struct CIOPurchaseItem {
```

Comment:
I'd rather we go with a more generic `CIOItem` class name, so we can potentially use it in like items field for recommendation event or browse event `items` field as well

Reply:
@crgee1 That's a good point. I had it as `CIOItem` originally but changed it to `CIOPurchaseItem` to be a little more consistent with how the Android repo has it set up (`PurchasItem`).

I'll switch it back to `CIOItem`, the only thing that concerns me a little is the quantity field, but since it's an optional parameter that's only used by purchase events I think it should be okay.